sdk/search/azure-search/src/main/java/com/azure/search/models/NamedEntityRecognitionSkill.java

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+package com.azure.search.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import java.util.List;
+
+/**
+ * Text analytics named entity recognition. This skill is deprecated in favor
+ * of EntityRecognitionSkill.
+ */
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "@odata.type")
+@JsonTypeName("#Microsoft.Skills.Text.NamedEntityRecognitionSkill")
+@Fluent
+public final class NamedEntityRecognitionSkill extends Skill {
```

Comment:
This skill doesn't work in api version 2019-05-06.  We probably still need to include it in case someone tries to use api version 2017-11-11-Preview, which is the only version that will technically let you use it, but just worth noting.

Reply:
@careyjmac Can you elaborate on what "doesn't work" means? I'm assuming you can't create the skill in new API versions, but if a skillset already exists that has the skill, can you GET that skillset from the REST API?

In general we're a bit stuck when it comes to supporting old stuff like this because as long as persisted resources exist that use them, we need the type for deserialization.

Reply:
Note from the code for this skill: 
// NOTE: This skill will be deprecated going forward 
    // i.e., for API-Versions that are newer than 2017-11-11-preview, we will not let customers create/update skillsets
    // with this skill. This class (and it's model copier remnants) have to remain in the codebase, until we decide to *deprecate and delete*
    // API-Version 2017-11-11-preview

Unfortunately you're probably right about needing to include the support, though I feel like if you can't create/update we should at least throw a warning or something when you do a get right?

Reply:
There's no way to throw a warning in HTTP. The GET request either works, or it doesn't. I think we're stuck with this until you can confirm that no persisted skillsets have this skill anymore.